.github/workflows/test.yml

```diff
@@ -0,0 +1,37 @@
+name: Test
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Run test over the merged changes
+        run: |
+            git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+            git config --global user.name "github-actions[bot]"
+            git fetch origin $GITHUB_BASE_REF
+            git checkout -b pr
+            git checkout $GITHUB_BASE_REF
+            git merge --no-commit --no-edit --allow-unrelated-histories pr
+            yarn
+            yarn test 
```

Comment:
we also forgot to add` yarn lerna run build` after `yarn`  😀

Otherwise tests will always fail